mods/cnc/rules/defaults.yaml

```diff
@@ -475,6 +475,8 @@
 	WithInfantryBody:
 		IdleSequences: idle1, idle2
 		StandSequences: stand, stand2
+	AttackFrontal:
+		FacingTolerance: 0
```

Comment:
Most of these yaml overrides should be able to go away now, right?

Reply:
Removed the superfluous yaml edits.